mobile/library/common/jni/BUILD

```diff
@@ -1,31 +1,52 @@
-load("//bazel:kotlin_lib.bzl", "envoy_mobile_so_to_jni_lib")
 load("//bazel:android_debug_info.bzl", "android_debug_info")
 load("@envoy//bazel:envoy_build_system.bzl", "envoy_package")
-load("//bazel:envoy_mobile_test_extensions.bzl", "TEST_EXTENSIONS")
 
 licenses(["notice"])  # Apache 2
 
 envoy_package()
 
+# Suport for JNI, using NDK on android and Java otherwise.
```

Comment:
it might be helpful to explain here, or elsewhere, why we use NDK on Android and "java" (not sure what java means in this context?) otherwise in order to get JNI functionality?  i didn't understand that part

Reply:
Ahh, that's great to know, thanks!  Probably can just add a very basic statement then like "using NDK on android and Java otherwise, since the JNI APIs differ a bit on Android JVMs" or something like that?

Reply:
My Android/Java expertise is limited, but Ryan's comment matches my understanding too. +1